cli/reference/blocks/stack.md

```diff
@@ -4,3 +4,44 @@ description: Learn how to configure stacks using the stack block.
 ---
 
 # The `stack` block
+
+Use the `stack` block to define a stack within a stack directory. Terramate detects stacks based on this `stack` block, and you can define only one `stack` block per stack directory. The stack metadata defined in the `stack` block is accessible only from the stack directory where it is defined. You can access this metadata via the `terramate` namespace using the `terramate.stack` object. For example, `terramate.stack.<key>` where `<key>` is any key defined in the `stack` block.
+
+## Arguments
+
+You can see `stack` metadata details [here](../../stacks/configuration.md#general-stack-metadata).
+
+- `id` (string) The user-defined id of the stack.
```

Comment:
Seems like we are missing arguments such as

- `before`
- `after`
- `wants`
- `wanted_by`
- `watch`

Are missing here - for details, please see https://terramate.io/docs/cli/stacks/configuration#after